chart/templates/rbac.yaml

```diff
@@ -3,93 +3,30 @@ kind: ClusterRole
 metadata:
   name: {{ .Release.Name }}
 rules:
-- apiGroups:
-    - management.cattle.io
-  resources:
-    - 'settings'
-  verbs:
-    - 'get'
-    - 'watch'
-    - 'list'
-- apiGroups:
-  - ""
-  resources:
```

Comment:
we are firing events, example: https://github.com/rancher/elemental-operator/blob/main/pkg/controllers/managedosversionchannel/managedosversionchannel.go#L57

Reply:
You are right, I have added it once again